release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -814,6 +814,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+ A default limit of 30% of the storage capacity of the node for the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```

Comment:
```suggestion
A default limit of 30% of the storage capacity of the node for the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```
- I think the blank space at the beginning of the line is causing the entire paragraph to render as a code block in the preview:

![image](https://github.com/openshift/openshift-docs/assets/86380613/cd0da874-14c8-4d59-8730-943b8bb252cf)

Reply:
Fixed!